models/agent_model.go

```diff
@@ -631,6 +638,18 @@ func (s Agent) TemplateDelimiters(svc *Service) *DelimiterPair {
 	return &tdp
 }
 
+func (s *Agent) buildWebConfigFile(password string) string {
+	buf, err := bcrypt.GenerateFromPassword([]byte(password), 14)
+	if err != nil {
+		log.Fatal(err, "cannot encrypt basic auth password")
```

Comment:
It is not `encryption` it is hashing. `cannot calculate hash for password`

Reply:
done